CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 All notable changes to this project will be documented in this file.
 
+# [v0.23.0](https://github.com/cryptape/cita-sdk-java/compare/v0.22.3...v0.23.0) (2019-04-30)
```

Comment:
Release date is 2019.4.30 ? If yes, please merge tomorrow.

Reply:
yes.I will merge tomorrow.